stripe/src/main/java/com/stripe/android/ApiVersion.kt

```diff
@@ -9,14 +9,26 @@ package com.stripe.android
  * See [https://stripe.com/docs/upgrades](https://stripe.com/docs/upgrades) for latest
  * API changes.
  */
-internal data class ApiVersion internal constructor(internal val code: String) {
+internal data class ApiVersion internal constructor(
+    internal val version: String,
+    internal var betas: Set<StripeApiBeta>? = null
```

Comment:
`Set<StripeApiBeta>` to be consistent with how you're doing it elsewhere? Also `val`?